Makefile

```diff
@@ -11,3 +11,9 @@ build-alpine:
 	docker build -f Dockerfile-alpine -t jakzal/phpqa:alpine .
 .PHONY: build-alpine
 
+
+NIGHTLY_TAG := jakzal/phpqa:nightly-$(shell date +%y%m%d)
+build-nightly:
+	docker build -t $(NIGHTLY_TAG) .
```

Comment:
Since the image was already built in the current job we could probably push that one instead of building it again.

Reply:
Perhaps creating a new tag and pushing it would work. The Current way does take ~3min for some reason.

Reply:
That's weird, it was quick for me. Of course provided other build steps were run before.